xray/core/plotting.py

```
@@ -0,0 +1,16 @@
+def _plot_dataarray(darray, *args, **kwargs):
+    """
+    Plot a DataArray
+    """
+    import matplotlib.pyplot as plt
+
```

Probably should raise an error if the data is not 1D